docs/markdown/Getting Started/getting-started/installation.md

```diff
@@ -6,40 +6,52 @@ createdAt: "2020-02-21T17:44:53.022Z"
 updatedAt: "2022-07-12T00:00:00.000Z"
 ---
 
-Most installations can use our one-step setup script to get a minimal version of Pants set up. If you need to make use of an advanced or non-standard setup, see our [manual installation](doc:manual-installation) page.
-
-Using our one-step setup script
--------------------------------
-
-Pants has a launch script (called `./pants`) that handles downloading, bootstrapping, and upgrading Pants, which you need to save at the root of your repository. 
-
-Pants also needs a `pants.toml` file, where you will eventually add all of the configuration needed to run testing, linting, and formatting rules. For now, it just needs to specify the version of Pants that you want to use. 
-
-To streamline this, we provide a script that will create a minimal `pants.toml` file specifying the latest released version of Pants, download, and run the `./pants` launch script.
+To install the `pants` binary use:
 
 ```
-/bin/bash -c "$(curl -fsSL https://static.pantsbuild.org/setup/one_step_setup.sh)" 
+/bin/bash -c "$(curl -fsSL https://static.pantsbuild.org/setup/pantsup.sh)" 
 ```
 
-If the installation process was successful, you will see `Pants was installed successfully!` echoed to your terminal. 
+This will install `pants` into `~/bin`, which must be on your PATH. The installer script will warn you if it is not.
 
-If you had difficulty installing Pants, see our [getting help](doc:getting-help) for community resources to help you resolve your issue.
+`pants` is a launcher binary that delegates to the underlying version of Pants in each repo. This allows you to have multiple repos, each using an independent version of Pants.
 
+- If you run `pants` in a repo that is already configured to use Pants, it will read the repo's Pants version from the `pants.toml` config file, install that version if necessary, and then run it.
 
-> 📘 Add `./pants` to version control
-> 
-> You should check the `./pants` script into your repo so that all users can easily run Pants.
+- If you run `pants` in a repo that is not yet configured to use Pants, it will prompt you to set up a skeleton `pants.toml` that uses that latest stable version of Pants. You'll then need to edit that config file to add [initial configuration](doc:initial-configuration).
+
+If you have difficulty installing Pants, see our [getting help](doc:getting-help) for community resources to help you resolve your issue.
 
 > 👍 Upgrading Pants
 > 
-> The `./pants` script will automatically install and use the Pants version specified in `pants.toml`, so upgrading Pants is as simple as editing `pants_version` in that file.
-
+> The `pants` launcher binary will automatically install and use the Pants version specified in `pants.toml`, so upgrading Pants in a repo is as simple as editing `pants_version` in that file.
+>
+> To upgrade the `pants` launcher binary itself, run
+> ```
+> SCIE_BOOT=update pants
```

Comment:
Is this the first reference to SCIE for a user? 

I think there needs to be a link to the scie-pants repo, to remove some ambiguity as to what this means.

Reply:
I don't think I agree. The user doesn't need more verbiage or terminology. They are just trying to GSD. We can treat this as a magic incantation. 

I can add a "If you're interested in how this works, here is the repo", but I don't want to introduce this as something you're expected to read or know about.

Reply:
Is `SCIE` an acronym that we even need to expose to the user? Why not an environment variable with "Pants" in the name?

Reply:
Or `pants self update` like `rustup self update` has.

Reply:
> "If you're interested in how this works, here is the repo"

I mean, at the very least, something like this - especially since the script is downloading a binary from the scie-pants repo, not this repo.

> Is SCIE an acronym that we even need to expose to the user? Why not an environment variable with "Pants" in the name?

Or if something like this happens. 

I'm personally very hesitant of one-line, bash-script installers - so to have that, and then have a strange acronym for the update path. As a first time user, that would feel weird without some context.

Reply:
I'd prefer not to expose SCIE to the user.  But the SCIE_BOOT functionality is in core scie-jump, which is unrelated to Pants. We could make that env var configurable I suppose, or implement indirection in scie-pants.

Reply:
FWIW: https://github.com/pantsbuild/scie-pants/issues/32

Reply:
@sureshjoshi you may be looking at this from the perspective of someone who wants to poke under the hood (as you indeed do). I think almost all users just want to get things working with the minimal amount of cognitive load. So I'm leery of introducing new concepts, repos, links etc.  I'd rather we implemented https://github.com/pantsbuild/scie-pants/issues/32 to get rid of the strange acronym, but in the short term saying "this is a magic incantation" seems OK? Explaining it adds cognitive load, and I'm not sure what is gained in exchange.

If you're hesitant of one-line bash installers, that's a separate issue, because that's 100% what this change is... What would your alternative be?

Reply:
I'm not a one-line bash person, and that's fine - won't ever be one. Many workarounds in my life (Brew, for reasons detailed in Slack and other). 

I'm generally not interested in poking under the hood, believe it or not - it's just when new, unknown terms are involved - I tend to like to see them defined rather than "trust us, this is the way". More of a good OSS citizen, rather than any actual interest in underlying process. 

I can't say it much better than John:

> The former leaks an implementation detail. A Pants user should not need to be confronted with what scie means. The latter is just plain clunky.

I agree that the linked issue is a better overall solution - and if that's the plan, then whatever comments I have about this page can be deferred to a later date to hopefully not need them.

Reply:
If we could get `pants` into standard brew (and apt) that would be great. But once you have to add a custom keg or whatever, the bash thing seems strictly easier to me...

So we're agreed on implementing https://github.com/pantsbuild/scie-pants/issues/32, and meanwhile I have added a "here's how this is implemented, but you don't need to know this" section.

Reply:
Could add as a trailing section, the option to install with brew?

Reply:
Definitely, now that I know it's a thing. Can you confirm the incantation? Post-rename?